docs/site/_includes/getting_started/existing/partials/TROUBLESHOOT_RU.liquid

```diff
@@ -0,0 +1,58 @@
+Кластеры некоторых провайдеров могут требовать дополнительных действий, необходимых для запуска Deckhouse и его компонентов (особенно при использовании managed-решений), например:
+    - узлы кластера могут иметь набор labels/taints которые не позволяют запускаться компонентам Deckhouse;
+    - в кластере могут работать validation-хуки, мешающие установке, и т.п.
+
+Ниже приведены некоторые такие случаи и частые проблемы. Если вы столкнулись с другими особенностями установки Deckhouse в существующем кластере, пожалуйста, опишите ваш опыт в [issue](https://github.com/deckhouse/deckhouse/issues).
+
+{% offtopic title="Ошибки на этапе 'Waiting for Deckhouse to become Ready'" %}
+- Ошибка вида:
+  ```
+  │ │ ┌ Waiting for Deckhouse to become Ready
+  │ │ │ Deckhouse pod found: deckhouse-64649df6f9-mf6dt (Pending)
+  │ │ │ Deckhouse pod found: deckhouse-64649df6f9-mf6dt (Pending)
+  ```
+
+  Возможная причина — в кластере нет узла с label `node-role.kubernetes.io/master: ""`, который первоначально используется в `nodeSelector` манифеста deployment `deckhouse`.
+
+  Варианты решения:
+  - Отредактировать deployment `deckhouse` и указать нужный nodeSelector:
+    ```shell
+    kubectl -n d8-system edit deployment/deckhouse
+    ```
+  - Удалить nodeSelector у deployment `deckhouse`:
+    ```shell
+    kubectl patch -n d8-system deployment deckhouse --type json -p '[{"op": "remove", "path": "/spec/template/spec/nodeSelector"}]'
+    ```
+{% endofftopic %}
+
+{% offtopic title="Кластер в VK Cloud Solutions (MailRu Cloud Solutions)" %}
+<ul><li><p>В кластерах версии 1.21+ присутствует Gatekeeper (OPA), который требует выставления limits и requests подам. Pod Deckhouse архитектурно не имеет limits/requests, и, после установки Deckhouse, его Pod не будет запущен. При просмотре событий Deployment <code>deckhouse</code> вы можете увидеть ошибку:</p>
+<div class="highlight"><pre><code>admission webhook "validation.gatekeeper.sh" denied the request: [container-must-have-limits] container <...> has no resource limits...</code></pre></div>
+
+<p>Чтобы Deckhouse смог запуститься необходимо удалить Gatekeeper. Для удаления Gatekeeper выполните на узле, имеющем доступ к API кластера, следующие команды (потребуется установка <a href="https://helm.sh/" target="_blank">helm-клиента</a>):</p>
+{% snippetcut selector="gatekeeper-uninstall" %}
+```shell
+helm delete gatekeeper --namespace opa-gatekeeper
+kubectl delete crd -l gatekeeper.sh/system=yes
+```
+{% endsnippetcut %}
+</li>
+<li>Для версий кластера 1.20+ необходимо удалить также <code>metrics-server</code>. Для удаления <code>metrics-server</code> выполните на узле, имеющем доступ к API кластера, следующие команды (потребуется установка <a href="https://helm.sh/" target="_blank">helm-клиента</a>):
+{% snippetcut selector="metrics-server" %}
+```shell
+helm -n kube-system uninstall metrics-server
```

Comment:
I think, we should add next information about "why user need to delete metrics server".

Because, Deckhouse use https://github.com/kubernetes-sigs/prometheus-adapter prometheus adapter and it conflict with metrics-server.

Reply:
@nabokihms fyi

Reply:
Oh my. When we spoke with Artem, it did not sound so awful as it actually looks on paper.
It seems it would be better to use another cloud provider-managed service as an example, e.g., EKS, AKS.

Reply:
The section was updated. Need review.